custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -76,38 +76,172 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
             locationLevelName = 'AWC';
             locationLevelNameField = 'awc_name';
         }
+        return {
+            'locationLevelName': locationLevelName,
+            'locationLevelNameField': locationLevelNameField
+        }
+    };
+
+    vm.sddTableData = {
+        'pw_lw_children': {
+            'non-awc' : [
+                {
+                    'mData' : 'num_launched_awcs',
+                    'heading' : 'Number of AWCs launched',
+                    'tooltipValue' : 'Total Number of Anganwadi Centers launched in the selected location.',
+                    'columnValueType' : 'raw',
+                    'columnValueIndicator' : 'num_launched_awcs',
+                    'stylingClass' : 'medium-col'
```

Comment:
nit: this appears to be the same on every item - should we just remove it and always assume the value or you expect it to change for some?

Reply:
Yeah!! I will remove it. As of now, it is pretty standard that this styling is followed for all the datatable columns